api/v2/guides/google-ss-integration.md

```diff
@@ -0,0 +1,81 @@
+
+# Google Ad Manager Secure Signals Integration Guide
+
+This guide covers integration steps for publishers using UID2 with the Google Ad Manager **secure signals** feature (previously known as **encrypted signals from publishers**, ESP). It includes the following sections:
+
+* [Overview](#overview)
+* [Allow Secure Signal Sharing](#allow-secure-signal-sharing)
+* [Publisher Integrations](#publisher-integration)
+* [Server-Only Integration](#server-only-integration)
+* [UID2 SDK Integration](#uid2-sdk-integration)
+* [Sample Applications](#sample-applications)
+
+>NOTE: To use the UID2 Google Ad Manager secure signals integration, if you are using an SDK you must have your UID2 integration already set up. This does not apply if you are using server-only integration. For a summary of all the integration options available, see [UID2 Integration Guides](README.md).
+
+## Overview
+
+Google secure signals is a way for publishers to pass "encrypted" user IDs to bidders that are approved by Google, via [Google Ad Manager](https://admanager.google.com/home/) and the [Google Ad Manager Ad Exchange (Adx)](https://support.google.com/admanager/answer/6321605?hl=en). The framework is an optional part of the Google Publisher Tag (GPT) library commonly used by publishers.
+
+With this framework, the following steps occur:
+
+1. Publishers push user ID signals (advertising tokens) to the secure signals feature.
+2. The secure signals feature caches them on the client side and then transparently passes them to Google Ad Manager.
+3. Google Ad Manager uses the UID2 tokens to make bid requests -- forwarding the tokens to approved bidders within Google Adx based on the publisher's preferences.
+
+## Allow Secure Signal Sharing
+
+For your Google Ad Manager account to be eligible to receive encrypted UID tokens, you must make sure that encrypted signals are properly shared with third-party bidders on your Google Ad Manager account.
+
+For details, see [Share encrypted signals with bidders](https://support.google.com/admanager/answer/10488752) (Google reference documentation) and then follow the steps in [Use a third-party signal provider](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/securesignals) to switch on UID2 as your signal provider.
+
+## Publisher Integration
+
+When an encrypted signal is cached, the secure signals feature does not execute the handler to generate a new signal. Because of this, it is necessary to clear the cache before login and after logout.
+
+Since the secure signals feature does not provide a way to delete or invalidate a specific ID, publishers must call the `window.googletag.secureSignalProviders.clearAllCache()` function to clear all shared encrypted signals as part of their login/logout workflows.
+
+The following is an example of calling the `window.googletag.secureSignalProviders.clearAllCache()` function:
+
+```
+window.googletag = window.googletag || { cmd: [] }; 
```

Comment:
Could we update this to 

`      window.googletag = window.googletag || { cmd: [] };
      window.googletag.cmd.push(function () {
        window.googletag.secureSignalProviders =
          window.googletag.secureSignalProviders || [];
        window.googletag.secureSignalProviders.clearAllCache();
      });`